### PR TITLE
Feat/21

### PR DESCRIPTION
**Pull request by VargaElod23**

### CODE REVIEW

**Review by Malak67 — APPROVED** [review]
Maybe it would be a good idea to add the styled components inside a subfolder of components (maybe called styled) to differentiate them from the reusable components. Another idea would be to add it in the name, ex: header.styled.ts. Just a thought.

**Comment by VargaElod23** [comment]
@farebord pls fix conflicts

**Comment by farebord** [comment]
> Maybe it would be a good idea to add the styled-components inside a subfolder of components (maybe called styled) to differentiate them from the reusable components. Another idea would be to add it in the name, ex: header.styled.ts. Just a thought.

@Malak67 I totally agree with this, and I think the original idea was to keep styled-components in the same file until they are used in another folder. 
If they become reusable we moved them into the components or styled folder.

**Comment by VargaElod23** [comment]
@farebord any update on this one?

**Comment by VargaElod23** [comment]
@farebord  The Rebase went sideways, I think. `root/client/` shouldn't exist, logic should be inside `services/client`. Also, I see that the page is created very specifically for the form that it handles. I would recreate it with much more attention to atomicity and reusability. Also, the form is not handled, please implement it with an actual login until the backend API is being called.

**Review by Malak67 — CHANGES_REQUESTED** [review]
@farebord Please fix the merge issue as per @VargaElod23 mentioned!

File: services/client/src/pages/AddHypePool/AddHypePool.tsx [hunk 1]
```diff
@@ -0,0 +1,203 @@
+import styled from 'styled-components';
+import TitleText from 'src/components/TitleText/TitleText';
+import Input from 'src/components/input/Input';
+import Button from 'src/components/button/Button';
+
+const Wrapper = styled.div`
```
Comment (Malak67) [inline services/client/src/pages/AddHypePool/AddHypePool.tsx:6]: One small thing, I would move all the styles into a separate file called AddHypePool.styled.tsx so we separate the logic.

Comment (farebord) [reply services/client/src/pages/AddHypePool/AddHypePool.tsx:6]: Sure! Working on that!

**Review by Malak67 — APPROVED** [review]

**Pull request merged**